hack/ci/cluster-components-install-upgrade.sh

```diff
@@ -24,7 +24,7 @@ main() {
     export DOCKER_TAG=${OVERRIDE_DOCKER_TAG:-${DOCKER_TAG}}
     export DOCKER_REPOSITORY=${OVERRIDE_DOCKER_REPOSITORY:-${DOCKER_REPOSITORY}}
     export PRINT_INSECURE_NOTES="false"
-    capact::install_upgrade::charts
+    capact::install
```

Comment:
I've hard-coded  `--version=@local` in the `capact install` command. You would need to adjust it to set `--version=@latest --helm-repo-url=@latest` Then it will use the latest version from the main branch.